source/simulator/assemblers/newton_stokes.cc

```diff
@@ -23,6 +23,7 @@
 #include <aspect/simulator_access.h>
 #include <aspect/utilities.h>
 #include <aspect/material_model/rheology/elasticity.h>
+#include <aspect/material_model/visco_plastic_simple.h>
```

Comment:
For the same reason as mentioned in the long comment on `newton.cc`, I think this is undesirable.